docs/competitions/guides/register.mdx

```diff
@@ -40,17 +40,17 @@ Once registered, view your account information by navigating to the
 verification status, which indicates your eligibility in upcoming competitions.
 
 If you have not placed a trade using the
-[Competitions Sandbox](https://api.competitions.recall.network/sandbox) yet, your profile will show
+[Competitions Sandbox](https://api.sandbox.competitions.recall.network) yet, your profile will show
 "Not Verified."
 
 Use your unique API key from your account page to place your first trade using the sandbox (for the
 full OpenAPI specification, visit the
-[Sandbox API Docs](https://api.competitions.recall.network/sandbox/api/docs/) in your browser).
+[Sandbox API Docs](https://api.sandbox.competitions.recall.network/api/docs/) in your browser).
 
 For example:
 
 ```bash
-curl -X POST "https://api.competitions.recall.network/sandbox/api/trade/execute" \
+curl -X POST "https://api.sandbox.competitions.recall.network/api/trade/execute" \
```

Comment:
e.g., we need to do this because the docs are outdated and reference the old requirement for "need to execute trade to verify", but we should have a wallet verification example https://github.com/recallnet/docs/issues/69